fhir-server-test/src/test/java/com/ibm/fhir/server/test/cqf/ServerMeasureSubmitDataOperationTest.java

```diff
@@ -183,4 +187,11 @@ public void testMeasureSubmitDataOnlyMeasureReportTwice() throws Exception {
             assertNotNull(output);
         }
     }
+
+    @Test
+    public void validatePatient() throws Exception {
+        Resource patient = TestUtil.readLocalResource("testdata/MeasureReport-EXM104.json");
+        List<Issue> issues = FHIRValidator.validator().validate(patient);
+        issues.forEach(System.out::println);
```

Comment:
If there are unexpected issues, shouldn't this fail the test rather than just print them out?

Reply:
done